src/game_api/game_manager.hpp

```diff
@@ -86,6 +86,25 @@ struct GameProps
     bool unknown9;
     bool unknown10;
     // there's more stuff here
+    std::array<size_t*, 12> unknown11; // pointers to something

```

Comment:
And this is probably pointers to the 12 possible inputs (4kb, 4xinput, 4dinput?), something like maybe

```c++
struct Input {
  bool enabled;
  // padding
  uint8_t input_index;
  // padding
  INPUTS input;
}
```